tests/run/py_ucs4_type.pyx

```diff
@@ -384,6 +383,14 @@ def uchar_cast_to_int(Py_UCS4 uchar):
     >>> uchar_cast_to_int(u'A')  # doctest: +ELLIPSIS
     Traceback (most recent call last):
     ValueError: invalid literal for int() with base 10: ...A...
+    >>> uchar_cast_to_int(u'²')  # doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    ValueError: invalid literal for int() with base 10: ...²...
+
+    # Verify against Python
+    >>> int(u'²')  # doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    ValueError: invalid literal for int() with base 10: ...²...
```

Comment:
I understand the consistency argument, but we really no longer need the `u` prefix.
```suggestion
    >>> uchar_cast_to_int('²')  # doctest: +ELLIPSIS
    Traceback (most recent call last):
    ValueError: invalid literal for int() with base 10: ...²...

    # Verify against Python
    >>> int('²')  # doctest: +ELLIPSIS
    Traceback (most recent call last):
    ValueError: invalid literal for int() with base 10: ...²...
```